src/components/EntityInfo/EntityMenu.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <menu id="entity-menu">
+    <!-- Staff Comments -->
+    <span v-if="!!businessId && isAllowed(AllowableActions.ADD_STAFF_COMMENT)">
+      <StaffComments
+        :axios="axios"
+        :businessId="businessId"
+        :maxLength="2000"
+      />
+    </span>
```

Comment:
Rather than writing conditionals for each allowable action, would it be possible to iterate over the allowable actions returned by the API and then use a `<component :is="">` to render these menu items dynamically?   It seems to me that this would make our code easier to extend and maintain.

Reply:
That's a good idea but I doubt it: the possible actions (and whether they're allowed or not) are sprinkled into different places in the code/components. It boils down to our desire to show a consistent page with various actions but those actions are in different places (header, right-side components, staff filings dropdown, etc).

This could be investigated further, but IMO it's not the highest priority in this UI, not even in the top 3-4 (ie, behind Filing History, Todo List, App.vue, local filings, and more).